charts/argo-cd/templates/argocd-configs/argocd-secret.yaml

```
@@ -34,9 +34,9 @@ data:
   {{- with .Values.configs.secret.gogsSecret }}
   webhook.gogs.secret: {{ . | b64enc }}
   {{- end }}
-  {{- with .Values.configs.secret.azureDevops }}
-  webhook.azuredevops.username: {{ .username }}
-  webhook.azuredevops.password: {{ .password | b64enc }}
+  {{- if and (.Values.configs.secret.azureDevops.username .Values.configs.secret.azureDevops.password) }}
```

```suggestion
  {{- if and .Values.configs.secret.azureDevops.username .Values.configs.secret.azureDevops.password }}
```

Otherwise `helm install` will complain `username is not a method but has arguments`

---

Wow 😇

---

I fixed in https://github.com/argoproj/argo-helm/pull/2443/commits/eb6fea326c4e18a0ae43022169c52dc3295a3dcb . 🙋